motionplan/planManager.go

```diff
@@ -58,6 +58,7 @@ type atomicWaypoint struct {
 	mp         motionPlanner
 	startState *PlanState // A list of starting states, any of which would be valid to start from
 	goalState  *PlanState // A list of goal states, any of which would be valid to arrive at
+	origGoal   bool       // If partial plans are requested, return up to the last explicit waypoint solved
```

Comment:
this variable is named confusingly. I'm not sure how this is different from `ReturnPartialPlan` in the opts

Reply:
Good point. I will update the comment to be much more clear